virttest/utils_net.py

```diff
@@ -4141,7 +4141,8 @@ def get_netkvm_param_value(vm, param):
             err = "Error occured when get value of %s. " % param
             err += "With status=%s, output=%s" % (status, output)
             raise exceptions.TestError(err)
-        value = output.strip().split('=')[1].strip()
+        lines = output.strip().split('\n')
+        value = lines[0].strip().split('=')[1].strip()
```

Comment:
We might use `splitlines()` here.

Reply:
Modified, pls review it again, thanks :)